packager/packager.js

```diff
@@ -169,7 +173,7 @@ function getDevToolsLauncher(options) {
       res.writeHead(200, {'Content-Type': 'text/html'});
       fs.createReadStream(debuggerPath).pipe(res);
     } else if (req.url === '/launch-chrome-devtools') {
-      var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
+      var debuggerURL = options.origin + ':' + options.port + '/debugger-ui';
```

Comment:
@frantic Yes, that's the idea. `options.origin` can be an ip address. This was suggested by @jsierles in https://github.com/facebook/react-native/issues/1429#issuecomment-106905639.